src/main/java/racingcargame/view/OutputView.java

```diff
@@ -0,0 +1,40 @@
+package racingcargame.view;
+
+import java.util.List;
+import racingcargame.domain.Car;
+
+public class OutputView {
+
+	public static final String RESULT_MESSAGE = "실행 결과";
+	public static final String WINNER_MESSAGE = "가 최종 우승했습니다.";
+	public static final int ZERO = 0;
+	public static final int ONE = 1;
+	public static final String RESULT_DELIMITER = " : ";
+
+	public void printResultMessage() {
+		System.out.println(RESULT_MESSAGE);
+	}
+
+	public void printResult(List<Car> cars) {
+		cars.forEach(car -> {
+			System.out.print(car.getName() + RESULT_DELIMITER);
+			printMovement(car.getMovement());
+		});
+		System.out.println();
+	}
+
+	public void printMovement(int movement) {
+		while (movement > ZERO) {
+			System.out.print("-");
+			--movement;
+		}
+		System.out.println();
```

Comment:
writeNewLine() 등의 private 메소드로 빼서 작성하면 깔끔할 것 같다는 개인 의견 남겨봅니다!
재사용하기도 쉬워질 것 같아요.

```suggestion
		writeNewLine();
		
		...
		
		private void wirteNewLine() {
		       System.out.println();
		}
```

Reply:
간단한 로직이지만 어떤 의도인지 분명하게 드러내 주는 방식인 것 같습니다! 적용해보겠습니다 ! 감사합니다